specs/mediawiki/v1/content.yaml

```diff
@@ -202,6 +221,19 @@ paths:
         # work in current Chrome and Firefox.
         # See also http://tools.ietf.org/html/rfc7231#section-7.1.2
         uri: /{domain}/sys/parsoid/html/{title}
+      x-amples:
+        - title: Should get a latest html of a page by title
+          request:
+            params:
+              title: Foobar
+            headers:
+              cache-control: no-cache
+          response:
+            status: 200
+            headers:
+              etag: /.+/
+              content-type: text/html
```

Comment:
Let's be precise, since we use regex notation elsewhere. The monitoring utility could (and probably should) interpret this as *I want the field to be exactly this* as-is.

>  in the test we check both a regexp and if the actual header starts with what's declared

Yup, but that's more of a utility on our end. The actual production monitor should be clear on what we mean where.